mapbox_gl_platform_interface/lib/src/method_channel_mapbox_gl.dart

```diff
@@ -141,7 +141,6 @@ class MethodChannelMapboxGl extends MapboxGlPlatform {
       OnPlatformViewCreatedCallback onPlatformViewCreated,
       Set<Factory<OneSequenceGestureRecognizer>>? gestureRecognizers) {
     if (defaultTargetPlatform == TargetPlatform.android) {
-      if (useHybridComposition) {
```

Comment:
if `useHybridComposition` is no longer needed, shouldn't the static variable itself and it's documentation with getters also be removed?

Reply:
You are right. I've forget it

Reply:
In addition to static `useHybridComposition` there is also a getter on Mapbox instance and some usages in the sample app